docs/conf.py

```diff
@@ -137,7 +137,7 @@ def doc_redirect_include(file_path):
     }
 }]
 
-autoapi_dirs = [os.path.join(PROJECT_ROOT, __meta__.__package__)]
+autoapi_dirs = [os.path.join(PROJECT_ROOT, __meta__.__package__), os.path.join(PROJECT_ROOT, "tests")]
```

Comment:
Tests should not be included as the package docs.
Only the actual classes/functions to implement functionalities should be documented.

Reply:
See previous [comment](https://github.com/Ouranosinc/cowbird/pull/35#discussion_r1229523735) about linking the tests and the docs. Waiting for a confirmation if I should just remove the doc reference I had added in the tests.